setup.cfg

```diff
@@ -41,7 +41,7 @@ packages = find:
 python_requires = >=3.7
 setup_requires =
 install_requires =
-    appdirs>=1.3.0
+    platformdirs>=2.5.0
```

Comment:
Is this min version? 2.5.0 was released a year ago, but older versions should work as well. This seems to be the bigger release:
https://github.com/platformdirs/platformdirs/releases/tag/2.1.0